game/scripts/vscripts/Sections/Chapter6/SectionOpening.ts

```diff
@@ -0,0 +1,280 @@
+            axe.SetAttackCapability(UnitAttackCapability.NO_ATTACK);
+            axe.FaceTowards(showcaseHeroLocation);
+        }),
+
+        // CW part
+        tg.immediate(_ => goalWatchCW.start()),
+        tg.spawnUnit(centaurName, showcaseHeroLocation, DotaTeam.GOODGUYS, centaurName, true),
+        tg.setCameraTarget(context => context[centaurName]),
+        tg.immediate(context => {
+            const centaur: CDOTA_BaseNPC_Hero = context[centaurName];
+            centaur.AddItemByName("item_blink");
+            centaur.AddExperience(30 - centaur.GetLevel(), ModifyXpReason.UNSPECIFIED, false, false);
+            const basicAbilities = [0, 1, 2].map(abilityIndex => getOrError(centaur.GetAbilityByIndex(abilityIndex)));
+            const ultAbility = getOrError(centaur.GetAbilityByIndex(5));
+            basicAbilities.forEach(ab => ab.SetLevel(4));
+            ultAbility.SetLevel(3);
+        }),
+        tg.wait(0.1),
+        tg.seq([
+            tg.wait(2),
+            castNoTarget(centaurName, 5),
+            tg.moveUnit(context => context[centaurName], showcaseHeroLocation.__add(Vector(200, 200)), true),
+            tg.immediate(context => {
+                const centaur: CDOTA_BaseNPC_Hero = context[centaurName];
+                const blink = getOrError(centaur.GetItemInSlot(0) as CDOTABaseAbility | undefined);
+                centaur.CastAbilityOnPosition(dummyLocation, blink, 0);
+            }),
+            tg.wait(0.5),
+            castNoTarget(centaurName, 0),
+            tg.wait(2),
+            castOnTarget(axeName, centaurName, 1),
+            tg.wait(5),
+            castOnTarget(axeName, centaurName, 1),
+            tg.wait(3),
+            tg.immediate(context => context[centaurName].RemoveSelf()),
+        ]),
+
+        tg.immediate(_ => goalWatchCW.complete()),
+        tg.audioDialog(LocalizationKey.Script_6_Opening_4, LocalizationKey.Script_6_Opening_4, ctx => ctx[CustomNpcKeys.SunsFanMudGolem]),
+
+
+        // PA part
+        tg.immediate(_ => goalWatchPA.start()),
+        tg.spawnUnit(paName, showcaseHeroLocation, DotaTeam.GOODGUYS, paName, true),
+        tg.setCameraTarget(context => context[paName]),
+        tg.immediate(context => {
+            const pa: CDOTA_BaseNPC_Hero = context[paName];
+            pa.AddItemByName("item_moon_shard");
+            pa.AddExperience(30 - pa.GetLevel(), ModifyXpReason.UNSPECIFIED, false, false);
+            const basicAbilities = [0, 1, 2].map(abilityIndex => getOrError(pa.GetAbilityByIndex(abilityIndex)));
+            const ultAbility = getOrError(pa.GetAbilityByIndex(5));
+            basicAbilities.forEach(ab => ab.SetLevel(4));
+            ultAbility.SetLevel(3);
+        }),
+        tg.wait(2),
+        tg.seq([
+            castNoTarget(paName, 2),
+            tg.wait(2),
+            castOnTarget(axeName, paName, 0),
+            tg.wait(0.5),
+            castOnTarget(axeName, paName, 1),
+            tg.wait(6),
```

Comment:
6 seconds sounds like a long time to wait without her doing much. Maybe alter the flow a little so there's no such a long downtime?

Reply:
That's just waiting for her cooldown, I thought the timing felt fine actually playing it

Reply:
Is she attacking? Couldn't figure that out, if she Phatom Strikes and shows off Coup de Grace in between that's fine.